Makefile

```diff
@@ -1,13 +1,15 @@
 include Makefile.defs
 
 SUBDIRS = envoy plugins bpf cilium daemon monitor cilium-health bugtool
-GOFILES ?= $(shell go list ./... | grep -v /vendor/ | grep -v /contrib/ | grep -v /test | grep -v cilium/envoy | grep -v envoy.*api)
+GOFILES ?= $(shell go list ./... | grep -v /vendor/ | grep -v /contrib/ | grep -v /test | grep -v envoy.*api)
 GOLANGVERSION = $(shell go version 2>/dev/null | grep -Eo '(go[0-9].[0-9])')
-GOLANG_SRCFILES=$(shell for pkg in $GOFILES; do find $(pkg) -name *.go -print; done | grep -v /vendor/)
-BPF_SRCFILES=$(shell find bpf/ -name *.[ch] -print)
+GOLANG_SRCFILES=$(shell for pkg in $(subst github.com/cilium/cilium/,,$(GOFILES)); do find $$pkg -name *.go -print; done | grep -v /vendor/)
```

Comment:
Probably out of scope for this PR but this might be simpler with `go list -f '{{ join .GoFiles " "}}' ./...`

Reply:
I tried this and it produces a list without any directory hierarchy: ```
can't load package: package github.com/cilium/cilium/common/tests: C source files not allowed when not using cgo or SWIG: bpf-event-test.c
cilium_client.go
daemon_client.go get_config_parameters.go get_config_responses.go get_debuginfo_parameters.go get_debuginfo_responses.go get_healthz_parameters.go get_healthz_responses.go patch_config_parameters.go patch_config_responses.go
...
```

Reply:
boo. Ignore me. I thought I'd tested it and it had fuller paths but I think there's some other template combination we need to throw in. Not worth it to investigate further.